client/src/components/Header/Header.js

```diff
@@ -32,6 +42,11 @@ const MenuButton = styled.button`
   border: none;
   position: absolute;
   left: 2 rem;
+  img {
+    position: ${({ open }) => (open ? "fixed" : "relative")};
```

Comment:
GJ 👍 
But why change it in the first place? You could give your image a position of fixed at all time or will that affect your layout?